nautobot/core/management/commands/build_ui.py

```diff
@@ -94,19 +94,24 @@ def run_command(self, command, message, cwd=settings.NAUTOBOT_UI_DIR):
             message (str): Message to display when command is executed
         """
         self.stdout.write(self.style.WARNING(message))
+        self.stdout.write(f"Running '{command}' in '{cwd}'...")
 
         try:
-            out = subprocess.run(
+            result = subprocess.run(
                 shlex.split(command),
                 check=False,
                 cwd=cwd,
                 env={**os.environ.copy()},
+                encoding="utf-8",
+                capture_output=True,
             )
         except FileNotFoundError as err:
-            raise CommandError(f"`{command}` failed with error: {err}")
+            raise CommandError(f"'{command}' failed with error: {err}")
```

Comment:
One thing of note is that this is what happens when `npm` isn't found, such as if it wasn't installed.